public/locales/en/translation.json

```diff
@@ -15,12 +15,21 @@
     "vertical_grid": "Vertical Grid",
     "horizontal_grid": "Horizontal Grid"
   },
+  "widget_customization_slider_settings": {
```

Comment:
This translation is not used at all. It should be used provided to `<LegendSettings />` component.

Reply:
That's correct. I need to confirm with Gosia if we need 'legend' or 'slider' as the position of the slider is also supported by the heatmap chart